cmd/xdpcap/main.go

```diff
@@ -108,7 +108,10 @@ func capture(flags flags) error {
 	go func() {
 		for {
 			pkt, err := filter.read()
-			if err != nil {
+			switch {
+			case err == errFilterClosed:
```

Comment:
You could also check perf.IsClosed here if you want to

Reply:
I thought it was neater to return a new `filter` specific error (that doesn't wrap the original `perf` error), checking `perf.IsClosed()` from `filter.read()` felt a bit leaky.